example/rbac/prometheus-operator/prometheus-operator-deployment.yaml

```diff
@@ -40,7 +40,6 @@ spec:
             memory: 100Mi
         securityContext:
           allowPrivilegeEscalation: false
-          readOnlyRootFilesystem: true
```

Comment:
According to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#securitycontext-v1-core, `readOnlyRootFilesystem` defaults to `false`.

Aren't we lowering the security bar here? :thinking:

Reply:
We’re still enforcing an entirely different set of things through PSPs/SCCs. I do agree that I wish this was available but I feel I’ve done the due diligence on figuring out why this disappeared, but I can’t figure it out. To be able to move forward for now I removed it.